client/src/shapes/shapes/rect/draw.ts

```diff
@@ -19,69 +19,108 @@ export const drawRectWithCtx =
 
     const centerX = normalizedAt.x + normalizedWidth / 2;
     const centerY = normalizedAt.y + normalizedHeight / 2;
-
     ctx.translate(centerX, centerY);
     ctx.rotate(rotation);
 
-    if (borderRadius === 0) {
+    const [topLeft, topRight, bottomRight, bottomLeft] =
+      typeof borderRadius === 'number'
+        ? [borderRadius, borderRadius, borderRadius, borderRadius]
```

Comment:
this may be a use case for a small utility function like `toBorderRadiusArray()` that takes border radius value and will always return a border radius array.